### PR TITLE
T3.1 — Slice 3 scaffold: SimCLR end-to-end on stub CSI

### LINKED ISSUE #50: T3.1 — Scaffold + SimCLR end-to-end with stub data
**Slice 3 — Collins Izuchukwu Okafor — Calibrated phase-noise injection → cross-chipset (or robustness) results**
**Tracer bullet T3.1**

> **Data format note.** Widar3.0 is consumed as raw Channel State Information from Intel 5300 NICs (the original `.dat` format), downloaded from IEEE DataPort to `data/widar3/raw/`. Each `.dat` parses via `csiread.Intel(...)` to a complex tensor of shape approximately `(T_packets, 30 subcarriers, 3 antennas)` per receiver. Sampling rate ~1000 packets/s; gesture instance ~1–2 s. See [`docs/slice-1-afk-plan.md`](../blob/main/docs/slice-1-afk-plan.md) section 6 for the full layout.

**Slice 3 data note.** Cross-chipset evaluation requires CSI-Bench (a different dataset, hosted on Kaggle), which is **not yet downloaded**. This slice ships in two stages: Stage A (now, on Widar3.0 raw CSI) prototypes the augmentation and reports robustness numbers on cross-subject test; Stage B (when CSI-Bench arrives) fits per-chipset noise models and reports cross-chipset transfer. The tracer bullets below cover Stage A; Stage B is filed as follow-up child issues.

## What this issue produces
Runnable end-to-end SSL pipeline under `src/slices/collins/` using stub raw-CSI data.

## Acceptance criteria
- [ ] `src/slices/collins/` created with `README.md`.
- [ ] Tiny CNN encoder, SimCLR loss, linear probe — per conventions doc 07.
- [ ] `run.py` runs end-to-end on stub `(1024, 30, 3)` data.
- [ ] CI green.

## Dependencies
- Blocked by: (none — first in slice)
- Blocks: #51 (T3.2)

## See also
- [`docs/08-team-work-plan.md`](../blob/main/docs/08-team-work-plan.md) Slice 3 section.
- [`docs/07-experiment-scaffold.md`](../blob/main/docs/07-experiment-scaffold.md) for project conventions.
- [`docs/slice-1-afk-plan.md`](../blob/main/docs/slice-1-afk-plan.md) section 6 for the raw-CSI dataset layout.
- [`docs/06-using-ai-well.md`](../blob/main/docs/06-using-ai-well.md) for AI-assisted-work standards.

### PR DESCRIPTION
**Pull request by c0llinx**
## What this does

Adds the Slice 3 scaffold under `src/slices/collins/`:

- `encoder.py` — `TinyCNN` (40,032 params, channels 32→64→128, kernel 3) + `(T,S,A)→(C,T)` reshape helper.
- `ssl.py` — `SimCLR`, `ProjectionHead`, `nt_xent_loss` (τ=0.5), `make_views` (augmentation hook), `pretrain_simclr`.
- `eval.py` — `linear_probe` on a frozen encoder via sklearn `LogisticRegression`, per docs/07.
- `data.py` — `StubCSI` returning `(T=1024, S=30, A=3)` random tensors with random labels in `{0..5}`, per the issue contract.
- `run.py` — end-to-end entrypoint; `python -m src.slices.collins.run` succeeds.
- `README.md` — slice plan, file layout, and an explicit Stage A / Stage B note for cross-subject (now) vs cross-chipset (future, pending CSI-Bench).
- `tests/slices/collins/test_smoke.py` — pytest assertion that the pipeline runs without crashing.

Mirrors George's T1.1 scaffold (`src/slices/george/`) file-for-file so the two slices are comparable. Doc-07 conventions followed throughout. Stub shape `(1024, 30, 3)` is the only deliberate departure from George's `(100, 30, 3)`, per #50's explicit data spec.

## Why

Closes #50. First tracer bullet of Slice 3.

## How I tested it

```
$ python -m src.slices.collins.run
[T3.1] encoder params: 40032
[T3.1] pre-train losses: [1.9391, 1.6347]
[T3.1] linear-probe accuracy on stub data: 0.125 (chance ~ 0.167)

$ pytest tests/slices/collins/ -v
tests/slices/collins/test_smoke.py::test_smoke_runs_end_to_end PASSED

$ black --check .   # 12 files, all clean
$ ruff check .      # all checks passed
```

NT-Xent loss decreases from 1.94 → 1.63 over 2 epochs on random data — the contrastive signal is alive (in-batch negatives push different stubs apart even with identity views). Linear-probe accuracy ≈ chance, as expected for noise input.

## Anything reviewers should pay attention to

- `make_views` takes an optional `augment_fn` hook. T3.5 (#54) will plug the phase-noise injection into this without touching `ssl.py`.
- Encoder param count is 40K, slightly under the doc-07 "~50K" target — matches George's choice for cross-slice consistency.
- README explicitly frames Stage A (Widar3.0 cross-subject robustness) vs Stage B (cross-chipset on CSI-Bench, future work). The slice's *headline causal claim* is Stage B; T3.6/T3.7/T3.8 deliver Stage A.

### CODE REVIEW

**Review by isongjosiah — APPROVED** [review]
LGTM
